README.md

```diff
@@ -17,11 +18,24 @@ Console.WriteLine($"{myStation} {nextArrival}"); //SANDY SPRINGS STATION 9 min
 ```
 
 ## Building
-```
-dotnet build
-```
 
-## Testing
-```
-dotnet test
-```
+[![Travis](https://img.shields.io/travis/syncromatics/Syncromatics.Clients.Marta.svg)](https://travis-ci.org/syncromatics/Syncromatics.Clients.Marta)
+[![NuGet](https://img.shields.io/nuget/v/Syncromatics.Clients.Marta.Api.svg)](https://www.nuget.org/packages/Syncromatics.Clients.Marta.Api/)
+[![NuGet Pre Release](https://img.shields.io/nuget/vpre/Syncromatics.Clients.Marta.Api.svg)](https://www.nuget.org/packages/Syncromatics.Clients.Marta.Api/)
+
```

Comment:
This project has some very basic steps for building. Would you mind adding the following under the badges in this section?

    This library is built using [.NET Core](https://www.microsoft.com/net/download). To build:

    ```
    dotnet build
    ```

    To run the tests:

    ```
    dotnet test
    ```